src/packages/@library/ui/tooltip/Tooltip.svelte

```diff
@@ -20,6 +26,11 @@
   let isDelayed = false;
 
   const toggleTooltip = () => {
+    if (title === "") {
+      show = false;
+    } else {
+      show = true;
+    }
```

Comment:
cant change `show` variable in library module, this should be handled from the app components only as a props, this variable holds some kind of accessibility for guest user.